### PR TITLE
Tkt rc

### PR DESCRIPTION
**Pull request by CatalinaAlbornoz**
### Before submitting

Please complete the following checklist when submitting a PR:

- [ ] All new features must include a unit test.
      If you've fixed a bug or added code that should be tested, add a test to the
      test directory!

- [ ] All new functions and code must be clearly commented and documented.
      If you do make documentation changes, make sure that the docs build and
      render correctly by running `make docs`.

- [ ] Ensure that the test suite passes, by running `make test`.

- [ ] Add a new entry to the `doc/releases/changelog-dev.md` file, summarizing the
      change, and including a link back to the PR.

- [ ] The PennyLane source code conforms to
      [PEP8 standards](https://www.python.org/dev/peps/pep-0008/).
      We check all of our code against [Pylint](https://www.pylint.org/).
      To lint modified files, simply `pip install pylint`, and then
      run `pylint pennylane/path/to/file.py`.

When all the above are checked, delete everything above the dashed
line and fill in the pull request template.

------------------------------------------------------------------------------------------------------------

**Context:**

**Description of the Change:**

**Benefits:**

**Possible Drawbacks:**

**Related GitHub Issues:**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,26 @@
...
+
+<p align="center">
+  <strong>This is a Quantum Carnival ticket! Can you collect them all?</strong>
+  <img src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/master/doc/_static/readme_image.png" width="300px">
+</p>
+
+<hr>
+
+<p align="center">
+>>>>>>> cb866797 (Added tkt to docs and readme)
```
Comment (antalszava) [inline README.md:54]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -32,6 +32,26 @@
 </p>
 
 <p align="center">
+<<<<<<< HEAD
+=======
```
Comment (antalszava) [inline README.md:36]:
```suggestion
```

File: doc/index.rst [hunk 3]
```diff
@@ -78,6 +78,22 @@ PennyLane Documentation
         </div>
     </div>
 
+Community
+---------
+We love having a diverse and engaged community that uses PennyLane and contributes to it. 
+We invite you to explore the different resources and events which are designed to help you learn 
+quantum computing and have fun in the process.
+
```
Comment (co9olguy) [inline doc/index.rst:86]:
```suggestion
```

**Review by antalszava — APPROVED** [review]
Looks good to me! :tada: :heart_eyes:

File: README.md [hunk 4]
```diff
@@ -31,6 +31,18 @@
...
+    <img width=100% src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/master/doc/_static/carnival_splash_bg.png">
+  </a>
+</p>
+
+<p align="center">
+  Announcing <strong>PennyLane Quantum Carnival</strong>. We're throwing a party to celebrate the 3rd anniversary of Pennylane's release! Throughout November, we will have special events, announcements, and releases, all happening under one big tent. <a href="https://pennylane.ai/carnival">Join the fun!</a>
+</p>
+
+<hr>
+
```
Comment (co9olguy) [inline README.md:45]: This PR will remove the ticket that was just added to the README. We'll need to `git rm` this page from this PR and make sure that when the release candidate is merged into master that it takes the readme from `master`

Comment (antalszava) [reply README.md:45]: Good catch, thank you :+1: I'll double-check that we're not updating the `README.md` file at the time we're merging the release candidate into `master`.

**Pull request merged**